news/2019-04-22-project-loved-week-of-april-21st.md

```diff
@@ -0,0 +1,237 @@
+
+[![](/wiki/shared/news/2019-04-22-project-loved-week-of-april-21st/mania/4-ln-dan-phase-ii.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[osu!mania 7K Dan Courses - LN Dan Phase II](https://osu.ppy.sh/beatmapsets/895138#mania) by [Jinjin](https://osu.ppy.sh/users/3360737) et al.  
+138 BPM, 8:43 | 7K 5.32★, 7K 5.63★, 7K 6.68★, 7K 7.42★, 7K 8.62★
+
+The time has come once again for another 7K Dan course to go up in Project Loved! This time, it is the second phase of the LN Courses. You know the drill, the charts go in the order of regular LNs, technical LNs, inverse LNs, and release LNs. This phase covers Dans 4 through 8, challenging mid- to high-tier LN players. Don't feel too bad if you don't immediately clear your target skill level; practice is key!
+
+[![](/wiki/shared/news/2019-04-22-project-loved-week-of-april-21st/mania/5-dichotomy-soft-mix.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[Haywyre - Dichotomy (Soft Mix)](https://osu.ppy.sh/beatmapsets/631054#mania) by [BilliumMoto](https://osu.ppy.sh/users/3862471)  
+160 BPM, 3:10 | 9K 1.74★, 9K 2.81★, 9K 3.59★, 9K 5.41★
+
+It's been ages since we had an anti-meta pick for Project Loved, let alone a 9K pick. This set has a lot of unique things to offer, ranging across full keysounds, an extensive spread, and proper SV usages.
+
+The set overall is particularly technical, as the keysounds do in fact match very well with the note placements. The top diff in particular is something to look out for, as it presents varying patterns that keep the chart very interesting. It has large chords, minijacks, bursts, and some amounts of LN usage. It's about time 9K had more exposure in Project Loved, and this mapset is something that everyone should try out!
+
+---
+
+From everyone on the Project Loved team, we hope you enjoy this week's selection, and look forward to the next! The winning beatmaps will hit the [Loved beatmap listing](https://osu.ppy.sh/beatmapsets?s=8) sometime next week and will be announced via [@osugame on Twitter](https://twitter.com/osugame).
```

Comment:
we hope you enjoyed

Reply:
🤔 are we expecting the readers to play every map and only then read the end of the post?

Reply:
this week's selection is related to the descriptions above, isn't it?
readers might enjoy the selection just from reading i think

Reply:
ok